src/atoms/dataEntry/FormSubTitle/FormSubTitle.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import React from 'react';
+import { Text } from '../../typography/Text';
+
+type SubTitleProps = {|
+  children?: string,
+  text?: string,
+  size?: PropSizes,
+|}
+
+const FormSubTitle = ({ children, text, size }: SubTitleProps) => (
```

Comment:
Where we should use this component?

Reply:
Why not?

Reply:
If we have `FormSubTitle` - where is `FormTitle`?